cglib/src/main/java/net/sf/cglib/util/ParallelSorter.java

```diff
@@ -224,7 +226,7 @@ public int compare(int i, int j) {
         public int compare(int i, int j) {
             float vi = a[i];
             float vj = a[j];
-            return (vi == vj) ? 0 : (vi > vj) ? 1 : -1;
+            return (floatEquals(vi, vj)) ? 0 : (vi > vj) ? 1 : -1;
```

Comment:
Can you please change the whole thing to `java.lang.Float#compare(float, float)` and `java.lang.Double#compare(double, double)`?